Cogs.Model/Property.cs

```diff
@@ -29,7 +29,16 @@ public bool IsPrimitive
                 return DataType.IsPrimitive;
             }
         }
-
+        /*
+        public int? MinLength { get; set; } = null;
+        public int? MaxLength { get; set; } = null;
+        public int? MinInclusive { get; set; } = null;
+        public int? MinExclusive { get; set; } = null;
+        public int? MaxInclusive { get; set; } = null;
+        public int? MaxExclusive { get; set; } = null;
+        public Enum Enumerations { get; set; } = null;
```

Comment:
We'll merge this, but I think we will have a List<string> for the accepted types. Enum would have to be known at compile time